component/service/src/main/java/org/exoplatform/social/rest/impl/space/SpaceRestResourcesV1.java

```diff
@@ -442,8 +442,8 @@ public Response updateSpaceById(@Context UriInfo uriInfo,
     if (space == null || (! spaceService.isManager(space, authenticatedUser) && ! spaceService.isSuperManager(authenticatedUser))) {
       throw new WebApplicationException(Response.Status.UNAUTHORIZED);
     }
-    if (space.getDisplayName().length() > 200 || !SpaceUtils.isValidSpaceName(space.getDisplayName())) {
-      throw new WebApplicationException(Response.Status.BAD_REQUEST);
+    if (model == null || model.getDisplayName() == null || model.getDisplayName().length() == 0 || model.getDisplayName().length() > 200 || !SpaceUtils.isValidSpaceName(model.getDisplayName())) {
```

Comment:
model == null was already checked above
model.getDisplayName().length() == 0 could be replaced with model.getDisplayName().isEmpty() or better you can use StringUtils.isBlank(model.getDisplayName()) to replace both checks